addons/common/slugify.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function slugify(text) {
+  return text.toString().toLowerCase()
+    .replace(/\s+/g, '-')           // Replace spaces with -
+    .replace(/[^\w\-]+/g, '')       // Remove all non-word chars
+    .replace(/\-\-+/g, '-')         // Replace multiple - with single -
+    .replace(/^-+/, '')             // Trim - from start of text
+    .replace(/-+$/, '');            // Trim - from end of text
+}
```

Comment:
I'm trying to figure out if I can avoid pulling in this guy: https://github.com/dodo/node-slug